doc-content/enterprise-only/optimizer/quarkus-creating-proj-proc.adoc

```diff
@@ -0,0 +1,93 @@
+[id='proc-quarkus-creating-proj_{context}']
+
+
+= Creating the school timetable project
+
+The school timetable project lets you get up and running with {PLANNER} and Quarkus application using Apache Maven and the Quarkus Maven plug-in.
+
+.Procedure
+
+. In a command terminal, enter the following command to verify that Maven is using JDK 11 and that the Maven version is 3.6 or higher:
++
+[source]
+----
+mvn --version
+----
+
+. If the preceding command does not return JDK {JDKVersion}, add the path to JDK {JDKVersion} to the PATH environment variable and enter the preceding command again.
+
+. To generate the project, enter one of the following commands:
++
+NOTE: Apple macOS and Microsoft Windows are not supported production environments.
++
+* If you are using Linux or Apple macOS, enter the following command:
++
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:1.11.6.Final:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=optaplanner-quickstart \
+    -Dextensions="resteasy,resteasy-jackson,optaplanner-quarkus,optaplanner-quarkus-jackson" \
+    -DnoExamples
+cd optaplanner-quickstart
+----
++
+These commands create the following elements in the  `./optaplanner-quickstart` directory:
++
+
+** The Maven structure
+** Example `Dockerfile` file in `src/main/docker`
+** The application configuration file
+
+. After the directory structure is created, open the `pom.xml` file in a text editor and examine the contents of the file to ensure it contains the following elements:
+
++
+[source,xml,subs=attributes+]
+----
+<dependencyManagement>
```

Comment:
Also please consider [2] in appendix of https://docs.google.com/document/d/1tmYhkpciXJW6ut6vDwBOD5ZxCZmBXHrxm5JEl4UB49E/edit#

The structure a bit different in the generated app. 
The dependency section:
the value of maven coordinates such as group id and artifact id and version are moved to property section

Also please change the version of it it should be :1.11.6.Final-redhat-00001 instead of 1.13.1.Final

Reply:
Fixed